terraform/application/application.tf

```diff
@@ -49,6 +49,8 @@ module "web_application" {
 
   replicas   = var.webapp_replicas
   max_memory = var.webapp_memory_max
+
+  enable_logit           = var.enable_logit
```

Comment:
no need for all the blank space (same for line 78)

Reply:
Fixed this.